### DISCUSSION TITLE
"View as" functionality for admins

### DISCUSSION THREAD

**Discussion by PhilippDehner** [grafana/grafana#38908] [General]
As known from other Platforms (e.g. discord) it would be really helpful for admins to use a function "view as". 

This allows an administrator to select a user / team and then temporarily see the dashbaords and functions that that particular role can see / do.

This would really help, currently admins have to create Test-Accounts to see, what some teams can do. If users have special rights, this method is not easy to use.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
